### PR TITLE
make ”Meet the Hoodie” box clickable

### PR DESCRIPTION
**Pull request by HipsterBrown**
Fix for issue #90 

Makes 'most' of the box clickable. It would be nice to add some type of :hover and :active transitions for increased interaction design.

### CODE REVIEW

File: dist/sass/_teaser-index.scss [hunk 1]
```diff
@@ -204,9 +204,14 @@ svg rect {
...
 
+            h3 {
+              color: #fff;
+            }
+
             a {
                 color: #fff;
-                margin-left: rem-calc(-15);
+                display: block;
+                //margin-left: rem-calc(-15);
```
Comment (verpixelt) [inline dist/sass/_teaser-index.scss]: I've double checked that. You can remove the margin declaration.

File: dist/sass/_typo.scss [hunk 2]
```diff
@@ -145,6 +145,10 @@ a {
     text-shadow: 1px 0 0 #fff, -1px 0 0 #fff;
 }
 
+a.no-line {
```
Comment (verpixelt) [inline dist/sass/_typo.scss]: Please use the already existing class `no-border`to remove the underline. (For writing selectors, please avoid combining element selectors with classes. We will update the readme with coding guidelines, soon.)

Comment (HipsterBrown) [inline dist/sass/_typo.scss]: Ok I'll make the change ASAP and re-submit the request. Is there an open issue for the coding guidelines for me to submit that addition as well?

**Comment by HipsterBrown** [comment]
Changes made. Any thoughts on :hover or :active interaction, or should I prototype and do a separate PR?

**Pull request merged**